problemsets-mat223/modules/module9.tex

```diff
@@ -71,7 +71,29 @@
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
 
-	XXX Finish
+	For $\mathcal T$:
+	
+	Let vectors $\vec u=\mat{u_1\\u_2},\vec v=\mat{v_1\\v_2}$ and scalar $\alpha$ be given. So $\vec u +\vec v=\mat{u_1+v_1\\u_2+v_2}$ and $a\vec u = \mat{a u_1\\a u_2}$. We then have 
+	\[
+	    \mathcal T(\vec u +\vec v)=\mat{2u_1+2v_1\\u_2+v_2}=\mat{2u_1\\u_2}+\mat{2v_1\\v_2}=\mathcal T(\vec u) + \mathcal T(\vec v)
+	\]
+	and
+	\[
+	    \mathcal T(a\vec u)=\mat{2a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal T(\vec u).
+	\]
+	Therefore, $\mathcal T$ is linear.
+	
+	For $\mathcal S$:
+	
+	We could easily find two counter examples such as:
```

Comment:
Let's leave the cleverness out of it. Also avoid saying things are "easy". Explain this one by starting naively, i.e., computing "S(\vec u+\vec v)$ and noticing that it doesn't look like $S(\vec u)+S(\vec v)$. "Therefore, we will guess that $S$ is not linear and look for a counter example. Using $\vec u=\mat{1\\0}$ and $\vec v=\mat{0\\1}$, we see ..."